addons/escoria-core/game/core-scripts/esc/commands/wait.gd

```diff
@@ -11,12 +13,12 @@ class_name WaitCommand
 func configure() -> ESCCommandArgumentDescriptor:
 	return ESCCommandArgumentDescriptor.new(
 		1, 
-		[TYPE_INT],
+		[TYPE_INT,TYPE_REAL],
```

Comment:
Please add a blank after the comma in the brackets here.

Reply:
Done